operator/risc_zero/lib/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "risc_zero_verifier_ffi"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+risc0-zkvm = { git = "https://github.com/risc0/risc0",  rev = "6107a65aa20816be9c62da0fa507794ebb1a682e" }
```

Comment:
we should create an issue or some reminder for us to check when the next release is available